lms/resources/_js_config/__init__.py

```diff
@@ -483,6 +487,7 @@ def _blackboard_sync_api(self):
                 },
                 "assignment": {
                     "resource_link_id": self._context.lti_params["resource_link_id"],
+                    "group_set_id": self._context.group_set_id,
```

Comment:
This whole block and the one above look very similar. It might be a good idea to have a function to produce the standard body, and then edit in the changes for the products. This has the nice effect of pointing out where they are similar or different.